tensorflow_io/core/kernels/orc/orc_kernels.cc

```diff
@@ -0,0 +1,215 @@
+#include <ctime>
```

Comment:
@oliverhu Can you add license header like other .cc files?

Reply:
sounds good. We can also add plugins to check license headers?